src/services/api.js

```diff
@@ -77,3 +77,13 @@ export function getGist(gist) {
       });
   });
 }
+
+export function version(serverUrl) {
+  return fetch(apiUrl(`/version?server_url=${serverUrl || ''}`))
+    .then(checkStatus)
+    .then(resp => resp.json())
+    .catch(err => {
+      console.error(err);
```

Comment:
`console` is defined. `err` is not. you can see comment about it in test. I can silent console.error by mocking console, but I'm not sure it's a good idea.

Reply:
you're right.
That's the problem with error messages in tests, where you only expect green lines.
Since it's not the scope of this PR, we could address this in the future

Reply:
I have created issue for it https://github.com/bblfsh/dashboard/issues/86